.github/workflows/run-post-submission-jobs.yaml

```diff
@@ -84,14 +158,6 @@ jobs:
         with:
           version: ">=0.0.1"
 
-      - name: Get nowcast date (aka round_id) 🕰️
```

Comment:
We can delete this step now that we're getting nowcast_date from the output of the new `get-all-dates` job.